packages/@molt/command/scripts/build-cjs.ts

```diff
@@ -1,81 +1,21 @@
 import { execaCommand } from 'execa'
```

Comment:
@rtbenfield I am trying out the emit of `package.json` with a `type: 'commons'` in this PR :)